cost-analyzer/values.yaml

```diff
@@ -362,7 +362,7 @@ kubecostFrontend:
   #   proxy_buffer_size   256k;
   #   large_client_header_buffers 4 64k;
   # hideDiagnostics: false # used if the primary is not monitored. Supported in limited environments.
-
+  # hideOrphanedResources: false # OrphanedResources is a primary-cluster-only report. WIP to support agent clusters.
```

Comment:
Something I learned (which will be enforced in #2734) is that comments on a line need to be separated from the value by 2 leading spaces, the `#`, and then one space. This is apparently "correct" YAML linting. I've already made those changes in the main `values.yaml` file.

Reply:
100% agreed!!!

Reply:
look right now?

Reply:
Yep, if my squinty vision does not deceive me.